### ISSUE TITLE
Receiving bindings

### ISSUE THREAD

**Issue by insolite** [marvinroger/node-rcswitch#11]
Hi! Are there any plans for implementing bindings for methods intended for receiving RF data? I mean [these ones](https://github.com/r10r/rcswitch-pi/blob/2c9c027217ae0b4fe132e0725db166ac29f93974/RCSwitch.h#L70-L80).

**Comment by slash-oleh** [comment]
Well, seems like [rcswitch-pi](https://github.com/r10r/rcswitch-pi) project (on which this project is based on) does not support code receiving. There are such function definitions but they do not seem to work (at least with [433Utils](https://github.com/ninjablocks/433Utils) example). Moreover, `rcswitch-pi` [is now deprecated](https://github.com/r10r/rcswitch-pi/issues/25) and is no longer supported. However the pure [rc-switch](https://github.com/sui77/rc-switch) now has a nice support of both RPI and receiving.
Summing up, @marvinroger would you mind if I create PR with replacing of `rcswitch` library with the latest version and add support of code receiving? I've already done some working prototype, just need to test things a bit.

**Comment by slash-oleh** [comment]
Done. I'd be really appreciate if someone would find time to review and release that.
P. S.
I guess `rcswitch` [source files](https://github.com/marvinroger/node-rcswitch/tree/master/externals) probably should be added using submodule or smth like that, but that's up to maintainer's choice of course.

**Comment by p33k** [comment]
+1

**Comment by marvinroger** [comment]
Oh, my bad, never noticed this PR! Thanks. 😉
